### ISSUE TITLE
Allow images to be marked as decorative

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov#506]
From @Gemma-inv : 

In my most recent accessibility user testing I spoke to a blind user about his frustration with unrelated image alt text breaking up the flow of focussing on content. With that in mind we are currently working with a customer to devise a user friendly way for content editors to label something as decorative - similar to the image example - Second option down 'choose an option' - 'Alt text' or 'Decorative Image'. Just wondering if anyone here has thoughts on this or has created something similar.

![image](https://user-images.githubusercontent.com/3852805/219640966-d2ee778b-78f2-475b-bc73-53a6269df7b2.png)
![image](https://user-images.githubusercontent.com/3852805/219641042-7fd53bad-b0c1-4d84-ba0d-0fb30123fd70.png)

**Comment by msayoung** [comment]
Copying in the relevant Slack chat :) 

Mark Conroy (Annertech)
  [32 minutes ago](https://localgovdrupal.slack.com/archives/C045Z40KGH0/p1676631634548709?thread_ts=1676628341.812099&cid=C045Z40KGH0)
This is a little tricky given how media is set up in Drupal. Since media is reusable, allowing the same image to be placed in more than one place on the site, an image could be decorative in one setting but part of the content in another.
We’d need to have a field beside each image field to say “in this instance, I want this image to be X or Y”. It’s not impossible.

Maria (Agile Collective)
  [21 minutes ago](https://localgovdrupal.slack.com/archives/C045Z40KGH0/p1676632313919199?thread_ts=1676628341.812099&cid=C045Z40KGH0)
That very point makes for a bit of an accessibility issue in itself really, as the alt text might change depending on the context that you are using the image. Go down the rabbit hole here: https://webaim.org/techniques/alttext/
Maybe we could set alt text on the media item, then override it when we use it somewhere (in the image paragraph). That would have the upside of forcing people to add alt text and then mark it as decorative in a considered way, rather than just leaving it blank.

Gemma Bates (Invuse)
  [12 minutes ago](https://localgovdrupal.slack.com/archives/C045Z40KGH0/p1676632830749949?thread_ts=1676628341.812099&cid=C045Z40KGH0)
I think the idea of making alt text required but marking as decorative on a case by case is a good solution.

**Comment by msayoung** [comment]
None of the below are the right solution, but are interesting nonetheless.

https://www.drupal.org/project/decorative_image_widget
https://github.com/c-e-mah/blur_no_alt

There are vibrant discussions happening around CKEditor 5 which is coming in Drupal 10:
https://www.drupal.org/project/drupal/issues/3222757#comment-14384063

And some interesting thoughts here too: https://accessibilitycluster.com/main-results/text-alternative-feature-%E2%80%93-presentation

**Comment by andybroomfield** [comment]
See also #394

**Comment by msayoung** [comment]
Flagging this again as it came up in the accessibility audit - specifically in relation to teasers in event listings and similar.
